core/src/main/scala/clue/TransactionalClientImpl.scala

```diff
@@ -16,7 +15,7 @@ import org.typelevel.log4cats.Logger
 //   "data": { ... }, // Typed
 //   "errors": [ ... ]
 // }
-class TransactionalClientImpl[F[_]: MonadError[*[_], Throwable]: TransactionalBackend: Logger, S](
+class TransactionalClientImpl[F[_]: MonadThrowable: TransactionalBackend: Logger, S](
```

Comment:
Is this from MTL?

Reply:
That's just a type alias for `MonadError[F, Throwable]`, so that it has a single type parameter. In Scala 2 we used kind-projector, but Scala 3 has its own syntax for this. So the type alias is to make it cross-compile.

I've since discovered that cats provides this, named `MonadThrow`, so I'll switch to that.